ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -88,9 +98,14 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 			<xs:pattern value="[+-]?[0-9]+(\.[0-9]+)?(h|m|s|ms)"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="limitedClockTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-9][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
+		</xs:restriction>
+	</xs:simpleType>
 	<xs:simpleType name="fullClockTimingType">
 		<xs:restriction base="xs:string">
-			<xs:pattern value="[0-9]+:[0-5][0-9]:[0-6][0-9](\.[0-9]+)?"/>
+			<xs:pattern value="[0-9][0-9]+:[0-5][0-9]:[0-6][0-9](\.[0-9]+)?"/>
```

Comment:
This permits values of seconds greater than 60, which there doesn't seem to be any spec support for. I think it should be `[0-9][0-9]+:[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?`.